corehq/apps/reports/static/reports/js/bootstrap5/maps_utils.js

```diff
@@ -256,8 +256,8 @@ hqDefine("reports/js/bootstrap5/maps_utils", [
             setMapHeight(map);
         };
         $(window).resize(resize);
-        $('#reportFiltersAccordion').on('shown', resize);
-        $('#reportFiltersAccordion').on('hidden', resize);
+        $('#reportFiltersAccordion').on('shown', resize);  // This ignores 'html_id_suffix'
```

Comment:
Was it not feasible to access the suffix here?

Reply:
I think I was unable to untangle the dependencies to `maps_utils.js` to ensure that "html_id_suffix" is always set. 87792d22abe untangles those missing dependencies. I've left a comment in the template.

(`maps_utils.js` is imported by `datatables_config.js`, which is used by hqadmin_base_report and fixtures, which I think is why I added `initial_page_data` to fixtures_base, although that was not necessary.)